src/components/ContentProfile/styled.ts

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components'
+
+export const Wrapper = styled.div`
+	width: 100%;
+	display: flex;
+	justify-content: center;
+`
+
+export const WrapperCenter = styled.div`
```

Comment:
Oh, and I'd maybe do some section or article or multiple of them instead of just styling divs. :thinking: Henry calls it "říše divů" when I nest too many of them. Just check what's recommended for HTML structures.

Reply:
Ok.